docs/configuration/elements.md

```diff
@@ -59,7 +59,8 @@ Global line options: `Chart.defaults.global.elements.line`.
 | `borderDashOffset` | `number` | `0.0` | Line dash offset. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineDashOffset).
 | `borderJoinStyle` | `string` | `'miter'` | Line join style. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineJoin).
 | `capBezierPoints` | `boolean` | `true` | `true` to keep Bézier control inside the chart, `false` for no restriction.
-| `fill` | <code>boolean&#124;string</code> | `true` | Fill location: `'zero'`, `'top'`, `'bottom'`, `true` (eq. `'zero'`) or `false` (no fill).
+| `cubicInterpolationMode` | `string` | `''` |  Interpolation mode to apply. [See more...](../charts/line.md#cubicinterpolationmode) 
```

Comment:
https://www.chartjs.org/docs/latest/charts/line.html#cubicinterpolationmode says that the valid options are `'default'` and `'monotone'`.  Should we either switch the default value here to `'default'` or there to empty string to make the docs in the two places match?

Reply:
The default is actually `undefined` (not an empty string) and [fallback to `not 'monotone'`](/chartjs/Chart.js/blob/b9290a20de0ac051b80eca3509163d597ecec476/src/controllers/controller.line.js#L283) in the line controller. I would maybe still use `'default'` as the default value instead of empty string since I find it less confusing.

Reply:
Sounds fine to me. @stockiNail can you update the default here to `'default'` for consistency?

Reply:
@simonbrunel if you go to Line chart documentation, you can see that for `cubicInterpolationMode` is written that the default is an empty string, for this reason I reported that value. Therefore also the documentation about Line chart should be changed, I guess.

<img width="576" alt="cubic" src="https://user-images.githubusercontent.com/11741250/55265312-11d16700-5278-11e9-8579-c9c28764c14f.png">

Reply:
@benmccann done!

Reply:
Ah, yeah I missed that https://www.chartjs.org/docs/latest/charts/line.html#cubicinterpolationmode and https://www.chartjs.org/docs/latest/charts/line.html#dataset-properties were different. We should probably put the same default value in all three places for consistency. Could you change the line chart one as well?

Reply:
updated line.md setting in the table as default undefined as it is described below on cubic interpolation description.